AFQ/tractography.py

```diff
@@ -57,11 +58,16 @@ def track(params_file, directions="det", max_angle=30., sphere=None,
         The miminal length (mm) in a streamline. Default: 10
     max_length: int, optional
         The miminal length (mm) in a streamline. Default: 250
+    verbose: bool, optional
+        If true, details about what the script is doing is printed. Default: False
 
     Returns
     -------
     list of streamlines ()
     """
+
+    if verbose:
+        print("    Loading Image...")
```

Comment:
Maybe use logging instead of `print` statements?

https://docs.python.org/3/library/logging.html